types/stoppable/stoppable-tests.ts

```diff
@@ -1,8 +1,11 @@
 import * as http from 'http';
 import stoppable = require('stoppable');
 
-const server: http.Server = stoppable(http.createServer());
-server.close();
+const server = stoppable(http.createServer());
```

Comment:
nit: Might be nice to type this to StoppableServer?

Reply:
Good suggestion, changes have been made.